### DISCUSSION TITLE
how to hide totals row

### DISCUSSION THREAD

**Discussion by elenaaralla** [6pac/SlickGrid#1035] [General]
Hi,

I'd like to hide the footer with totals on each group (or show it as header); is it possible?

![Schermata 2024-06-11 alle 17 46 36](https://github.com/6pac/SlickGrid/assets/14870650/a966063a-9f5b-4dad-bf96-edd68a4639e6)

**Comment by ghiscoding** [comment]
You should look at the available demos and try some of the options available in the grouping, the `aggregateCollapsed` might be what you're looking for. 

https://github.com/6pac/SlickGrid/blob/28113f4473c78fa9fccda7ade279d57c7b4b5d5e/examples/example-grouping-esm.html#L216-L227

As for the grouping, you can format it the way you want with the grouping `formatter` but the group only has a `value` (column group name) and a `count` (group items count) which will not be enough for your use case. You might be able to get the group totals via `dataView.getGrouping()` I'm not sure since I haven't tested it but if it works then you might want to call that in the group formatter that I mentioned above. You could also get the group totals if you know the row index then you can use `grid.getDataItem(idx)`... but I'm not even sure that would work because I think the group totals are shown at the bottom simply because it needs to aggregate them before showing them (hence why it's at the bottom), so it's possible that it's not possible at all to show these totals in the group (top) because it's too early and the totals at that time might be 0 (unless that is what `lazyTotalsCalculation` is for, I'm not sure).

Side note, it would be better if you ask these kind of questions on Stack Overflow, thank you.
